docs/release-notes/release-notes-0.18.0.md

```diff
@@ -259,6 +259,9 @@
   interface](https://github.com/lightningnetwork/lnd/pull/7736) to simplify the
   code. 
 
+* [Correct `fmt.Errorf` error wrapping 
+  instances](https://github.com/lightningnetwork/lnd/pull/8503)
+
```

Comment:
Do we want this in the release notes?

Reply:
Could've added the `no-changelog` label to avoid the CI fail, but fine as is IMO.